questions/shop.md

```diff
@@ -0,0 +1,165 @@
+    tag @a remove buyer.netherite
+    
+Or if you prefer a function:
+
+    # function example:load
+    scoreboard objectives add diamonds dummy
+    
+    # function example:buy/netherite
+    execute store result score @s diamonds if items entity @s container.* diamond
+    execute if entity @s[scores={diamonds=5..}] run function example:buy/netherite/success
+    execute unless entity @s[scores={diamonds=5..}] run tellraw @s {"text":"You don't have 5 diamonds","color":"dark_red"}]
+    
+    # function example:buy/netherite/success
+    give @s netherite_ingot 1
+    clear @s diamond 5
+    tellraw @s {"text":"You bought a netherite ingot for 5 diamonds","color":"green"}
+
+> [!NOTE]
+> The function `example:buy/netherite` must be run `as` the player
+
+### Bedrock
+In bedrock we have the `hasitem` argument, so it uses less commands than in Java.
+
+#### With an NPC
+This commands must be run in this order in an NPC
+
+    tag @initiator[hasitem={item=diamond,quantity=5..}] add buy.netherite
+    clear @initiator[tag=buy.netherite] diamond 5
+    give @initiator[tag=buy.netherite] netherite_ingot 1
+    tellraw @initiatior[tag=buy.netherite] {"rawtext":[{"text":"§2You bought a netherite ingot"}]}
+    tellraw @initiator[tag=!buy.netherite] {"rawtext":[{"text":"§3You don't have 5 diamonds"}]}
+    tag @initiator[tag=buy.netherite] remove buy.netherite
+
+_Related: [How to setup a NPC?](wiki/questions/npc)_
+
+#### Without an NPC
+If you don't want to use an NPC you canuse this method, is very similar to Java but it uses the `hasitem` argument instead.
+
+    tag @p add buyer.netherite
+    give @a[tag=buyer.netherite,hasitem={item=diamond,quantity=5..}] netherite_ingot
+    tellraw @a[tag=buyer.netherite,hasitem={item=diamond,quantity=5..}] {"rawtext":[{"text":"§2You bought a netherite ingot"}]}
+    tellraw @a[tag=buyer.netherite,hasitem={item=diamond,quantity=..5}] {"rawtext":[{"text":"§3You don't have 5 diamonds"}]}
+    clear @a[tag=buyer.netherite,hasitem={item=diamond,quantity=5..}] diamond 5
+    tag @a remove buyer.netherite
+
+### Add more than one items
+In this gide we will use just one item, but you can have multiples but it will require a second tag, that must be added if the player has buth items.
+Then when we clear the items we are going to clear them for the player with that tag.
+
+In this example we will buy a gold block with 2 emeralds and 5 diamonds. If you are in Java you will need one scoreboard for each item, assuming you already store the items result in it.
+
+    # Example
+    tag @p add buyer.example
+    execute as @a[tag=buyer.example] run tag @s[scores={diamonds=5..,emeralds=2..}] add buy.example
+    # run any tellraw to the player with the tag buy.example
+    clear @a[tag=buy.example] diamond 5
+    clear @a[tag=buy.example] emerald 2
+    give @a[tag=buy.example] gold_block 1
+And then we remove all the previus used tags:
+
+    tag @a remove buy.example
```

Comment:
duplicated line. I assume one of them was supposed to say "buyer.example"?

Reply:
You assumed correctly